README.md

```diff
@@ -5,8 +5,69 @@
 1. Write SQL code in your text editor instead of your command line. 
 2. Learn how to execute SQL code you've written in your text editor against a database you've created. 
 
-## Notes for whoever writes this:
+## Writing SQL in a Text Editor
 
-Explain how to create a `.sql` file that contains SQL code. Explain how to execute that file against a database in the command line. Then, walk through opening up that database and running a query that will confirm that the SQL in the `.sql` file was actually executed. 
+Up until now, we've been executing our SQL commands directly in the terminal. It is likely, however, that you will find yourself writing SQL in a file and executing that file in the context of your database. The more complex our databases become, the more tables we add and the more advanced the queries we run against them, the harder it will become to keep track of it all in the `sqlite3` prompt in our terminal. 
 
-Use the text from [this section](https://github.com/learn-co-curriculum/database-basics#advanced-writing-sql-in-a-text-editor) of the Database Basics read me to inform this reading. Read the [issues on this repo](https://github.com/learn-co-curriculum/database-basics) to do so. Then, remove the ["advanced" section] (https://github.com/learn-co-curriculum/database-basics#advanced-writing-sql-in-a-text-editor) from the Database Basics reading. This reading will directly follow Database Basics. 
+SQL is a programming language like any other, so we can write SQL in our text editor and execute it. This allows us to keep better track of our SQL code, including the SQL statements that create tables and query data from those tables. 
+
+To write SQL in our text editor and execute that SQL against a specific database, we'll create files in our text editor that have the `.sql` extension. These files will contain valid SQL code. Then, we can execute these files *against our database* in the command line. We'll take a look at this process together in the following code along. 
+
+## Code Along
+
+### Creating a Database and Table
+
+1 . In the terminal, create a database with the following command: 
+
+`sqlite3 pets_database.db`
+
+**Once you create your database, exit the sqlite prompt with the `.quit` command.**
+
+Open up a text editor (such as Sublime Text) and create and save a file `01_create_cats_table.sql`. In this file, write your create statement:
+
+```sql
+CREATE TABLE cats (
+  id INTEGER PRIMARY KEY,
+    name TEXT, 
+    age INTEGER
+);
+```
+
+2 . Execute that file in the command line. *Before running the below command, make sure that you've exited the sqlite prompt that you were in earlier when you created the database.* 
+
+`sqlite3 pets_database.db < 01_create_cats_table.sql`
```

Comment:
fixed with a note on deleting the cats table if you have a cats table hanging around from a previous exercise.